### ISSUE TITLE
Image viewer is really pixellated and ugly

### ISSUE THREAD

**Issue by rralian** [wordpress-mobile/WordPress-Android#2591]
Testing on a nexus 7 device with android 5.1, app 4.0-rc-2. This is what the image viewer looks like. I think it's the fact that the images are coerced/skewed to a landscape format.

![first image](https://cloudup.com/cUatwq9rPA2+)

![second image](https://cloudup.com/cZwDguskwLl+)

**Comment by rachelmcr** [comment]
I tested this again and reproduced the issue. When I use "Multi-select with the new picker" to select the image, I get the same results as above. I also see pixellation when I use "Select a photo from gallery" and "Select from media library" even though those images are displayed in a square and not a landscape format.

Tested on Nexus 7, Android 5.1.1, app beta 4.8-rc-3 and Nexus 9, Android 6.0, app beta 4.8-rc-3.
